mailpoet/lib/WP/Functions.php

```diff
@@ -563,6 +563,10 @@ public function wpLoginUrl($redirect = '', $forceReauth = false) {
     return wp_login_url($redirect, $forceReauth);
   }
 
+  public function wpNonceTick() {
+    return wp_nonce_tick();
+  }
+
```

Comment:
This doesn't seem to be used, if I'm not mistaken.

Reply:
Good catch, thanks! I first tried to set the transient expiration based on this value before realizing it wasn't the correct thing to use.